### ISSUE TITLE
Dead link in Week 3, Day 4 Gradient Descent section

### ISSUE THREAD

**Issue by ry-v1** [dphi-official/five-week-data-science-bootcamp#4]
Hello Team, 

There is a dead link in Week 3, Day 4 Gradient Descent section. Link is https://mc.ai/a-dummies-guide-to-gradient-descent-and-backpropagation/
